include/uapi/linux/if_link.h

```diff
@@ -984,6 +984,9 @@ enum {
 	IFLA_XDP_DRV_PROG_ID,
 	IFLA_XDP_SKB_PROG_ID,
 	IFLA_XDP_HW_PROG_ID,
+#ifdef CONFIG_XDP_LUA
+	IFLA_XDP_LUA_PROG,
```

Comment:
I believe we should uniformize our "namespace".. it looks like we should use `XDP_LUA` everywhere instead of `XDPLUA`, right?